### PR TITLE
Add missing ports to istio reseved ports list

### PR DESCRIPTION
**Pull request by AlonaKaplan**
**What this PR does / why we need it**:
istio documentation specify the list of ports istio is using - https://istio.io/latest/docs/ops/deployment/requirements/#ports-used-by-istio

kubevirt is using this list to avoid forwarding traffic to the vm on those ports in case of masquerade binding. And to avoid opening socket on those ports in case of passt.

Some ports were missing in the list kubevirt is using, this PR adding the missing ports.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by xpivarc — COMMENTED** [review]
/approve

**Review by phoracek — APPROVED** [review]

**Review by phoracek — APPROVED** [review]

**Comment by xpivarc** [comment]
/hold 
Helping Tide.
The storage lane is currently broken. https://github.com/kubevirt/kubevirt/pull/9135 should fix this.
Feel free to remove the hold once this is merged.

**Pull request merged**
